Failed to install llvm-node on macOS Sierra

OS version: macOS 10.12.4
LLVM version: Homebrew installed LLVM 4.0.0_1

Below is the error message when `npm install llvm-node`:

```
> llvm-node@1.0.5 install /Users/linjie/Desktop/t/node_modules/llvm-node
> node-gyp rebuild

  CXX(target) Release/obj.target/llvm/src/llvm-node.o
  CXX(target) Release/obj.target/llvm/src/bitcode/bitcode.o
  CXX(target) Release/obj.target/llvm/src/bitcode/bitcode-writer.o
  CXX(target) Release/obj.target/llvm/src/ir/ir.o
  CXX(target) Release/obj.target/llvm/src/ir/attribute.o
  CXX(target) Release/obj.target/llvm/src/ir/alloca-inst.o
  CXX(target) Release/obj.target/llvm/src/ir/array-type.o
  CXX(target) Release/obj.target/llvm/src/ir/argument.o
../src/ir/argument.cc:69:26: error: no matching constructor for initialization of 'llvm::Argument'
    auto* argument = new llvm::Argument { type, name, function, argNo };
                         ^              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/local/Cellar/llvm/4.0.0_1/include/llvm/IR/Argument.h:46:12: note: candidate constructor not viable: requires at most 3 arguments, but 4 were provided
  explicit Argument(Type *Ty, const Twine &Name = "", Function *F = nullptr);
           ^
/usr/local/Cellar/llvm/4.0.0_1/include/llvm/IR/Argument.h:34:7: note: candidate constructor (the implicit copy constructor) not viable: requires 1 argument, but 4 were
      provided
class Argument : public Value, public ilist_node<Argument> {
      ^
../src/ir/argument.cc:120:15: error: no member named 'addAttrs' in 'llvm::Argument'
    argument->addAttrs(builder);
    ~~~~~~~~  ^
2 errors generated.
make: *** [Release/obj.target/llvm/src/ir/argument.o] Error 1
```

--- a/src/ir/argument.cc
+++ b/src/ir/argument.cc
@@ -63,7 +63,7 @@ NAN_METHOD(ArgumentWrapper::New) {
         argNo = Nan::To<uint32_t>(info[3]).FromJust();
     }
 
-#if __clang_major__ == 4
+#if LLVM_VERSION_MAJOR == 4
     auto* argument = new llvm::Argument { type, name, function };
 #else
     auto* argument = new llvm::Argument { type, name, function, argNo };
@@ -113,13 +113,12 @@ NAN_METHOD(ArgumentWrapper::addDereferenceableAttr) {
     llvm::AttrBuilder builder {};
     builder.addDereferenceableAttr(bytes);
 
-#if __clang_major__ == 4
+#if LLVM_VERSION_MAJOR == 4
     auto attributes = llvm::AttributeSet::get(argument->getContext(), argument->getArgNo() + 1, builder);
     argument->addAttr(attributes);
 #else
     argument->addAttrs(builder);
 #endif
-
 }
 
 llvm::Argument *ArgumentWrapper::getArgument() {

